src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -93,13 +93,27 @@
 				{{ t('spreed', 'Delete conversation') }}
 			</NcActionButton>
 		</template>
+		<template v-else-if="item.token" slot="actions">
```

Comment:
**suggestion (non-blocking):** special slot attribute syntax is deprecated

 ```suggestion
		<template v-else-if="item.token" #actions>
```

Reply:
Same for other 3 slots in this component

Reply:
It was in `NcAppSidebar` and `NcAppSidebarTab`